### PR TITLE
Add RDS event subscriptions

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: resources/rds_event_subscriptions.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+func RdsEventSubscriptions() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_rds_event_subscriptions",
+		Description:  "Contains the results of a successful invocation of the DescribeEventSubscriptions action.",
+		Resolver:     fetchRdsEventSubscriptions,
+		Multiplex:    client.AccountRegionMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Columns: []schema.Column{
```
Comment (roneli) [inline resources/rds_event_subscriptions.go:22]: missing PK

**Review by roneli — CHANGES_REQUESTED** [review]
Missing PK, we should also want to add some e2e testing for this resource

**Review by amanenk — APPROVED** [review]
Looks good.

**Comment by irmatov** [comment]
e2e test added

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**